ansys/api/modelcenter/v0/workflow_messages.proto

```diff
@@ -453,3 +453,25 @@ message MoveComponentRequest {
    */
   int32 index_parent = 3;
 }
+
+/**
+ * Represents a request to resize a reference array.
+ */
+message SetReferenceArrayLengthRequest {
+  /**
+   * The reference array to modify.
+   */
+  ansys.api.modelcenter.v0.element_messages.ElementId target = 1;
+
+  /**
+   * The new size of the array.
+   */
+  ansys.api.modelcenter.v0.variable_value_messages.IntegerValue new_size = 2;
```

Comment:
Why not just use one of gRPC's built in integer primitives here?

Reply:
Hm, I was just mirroring the get size method, but I could switch it if we don't think it makes sense here.

Reply:
But here you are already in a message type, so you're free to use the primitives if you want.